src/game/Spells/SpellEffects.cpp

```diff
@@ -5638,13 +5629,10 @@ void Spell::EffectActivateObject(SpellEffectIndex eff_idx)
             break;
         case GameObjectActions::AnimateCustom0:
             gameObjTarget->SendGameObjectCustomAnim(0);
-            break;
```

Comment:
wtf are you doing here, now it will fall through

Reply:
was just a test, i fixed it via spell_effect_mod